integration-tests/kubernetes/quarkus-standard-way/src/test/resources/kubernetes-with-rbac-full.properties

```diff
@@ -15,4 +15,8 @@ quarkus.kubernetes.rbac.service-accounts.user.namespace=projectc
 quarkus.kubernetes.rbac.role-bindings.my-role-binding.subjects.user.kind=ServiceAccount
 quarkus.kubernetes.rbac.role-bindings.my-role-binding.subjects.user.namespace=projectc
 quarkus.kubernetes.rbac.role-bindings.my-role-binding.role-name=pod-writer
-quarkus.kubernetes.rbac.role-bindings.my-role-binding.cluster-wide=false
\ No newline at end of file
+quarkus.kubernetes.rbac.role-bindings.my-role-binding.cluster-wide=false
```

Comment:
Not a change request, but more of a recommendation: As kubernetes integration tests increase in number its getting harder to spot the matching property file. It's so much easier when config is provided in the java code using `overrideConfigKey` in the `QuarkusProdModeTest`.